packages/bundler-sdk-viem/src/types/operations.ts

```diff
@@ -99,33 +101,54 @@ export type Erc20BundlerOperations = {
 export type Erc20BundlerOperation =
   Erc20BundlerOperations[Erc20BundlerOperationType];
 
+export const VAULTV2_BUNDLER_OPERATIONS = [
```

Comment:
```suggestion
export const VAULT_V2_BUNDLER_OPERATIONS = [
```

Because we write `VaultV2` 😬😬